### PR TITLE
[ADD] ecommerce: new payment providers page

### PR DESCRIPTION
**Pull request by toaa-odoo**
Adding a documentation on payment providers specifically for the ecommerce section.

taskid: 3138597

### CODE REVIEW

**Review by jaae-odoo — APPROVED** [review]

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+=================
+Payment providers
+=================
+
+The default payment method for the eCommerce app is the **wire transfer**. Although not mandatory,
+it is strongly recommended to add additional payment providers for your customers. Odoo supports a
+multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
+for your website.
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:8]:
```suggestion
The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,
adding additional payment methods for your customers is strongly recommended. Odoo supports a
multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
for your website.
```
- wire transfer (I think no _the_ before)
- adding additional payment **methods**

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.. seealso::
+   :doc:`../../../../applications/finance/payment_providers`
+
+Configuration
+-------------
+
+To setup payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:16]:
```suggestion
To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->
```
[Setup is one word when it is a noun or an adjective. It is two words when it functions as a verb.](https://grammarist.com/spelling/set-up-vs-setup/)

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Payment Providers`. From here, :guilabel:`Activate` the payment providers you wish to have available
+on your shop, and configure them according to your needs.
+
+Alternatively, you can also access **payment providers** via :menuselection:`Website -->
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:20]:
```suggestion
Alternatively, you can access **payment providers** via :menuselection:`Website -->
```

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+the :guilabel:`Authorize.net` payment provider, the :ref:`Payment Capture Method
+<payment_providers/features/manual_capture>` can be configured in that same menu.
+
+If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure
+the :guilabel:`Paypal - Express Checkout` feature in this menu.
+
+Checkout payment options
+------------------------
+
+Once activated, customers can choose the payment provider of their choice during the **checkout
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:32]: payment method?

**Review by xpl-odoo — CHANGES_REQUESTED** [review]

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst (hunk 4 above)
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:32]: After checking with @AntoineVDV, it is indeed a payment provider. Might change in the future :)

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Alternatively, you can access **payment providers** via :menuselection:`Website --> Configuration
+--> Settings`. In the :guilabel:`Shop - Payment` section, you can :guilabel:`Configure SEPA Direct
+Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
+:guilabel:`Authorize.net` payment provider, the
+:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
+same menu.
+
+If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure
+the :guilabel:`Paypal - Express Checkout` feature in this menu.
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:28]: The "Paypal express checkout" feature doesn't exist (yet).

@pierre-pde You promised to remove it!

File: content/applications/websites/ecommerce/checkout_payment_shipping.rst [hunk 6]
```diff
@@ -0,0 +1,10 @@
+:nosearch:
+
+===============================
+Checkout, payment, and shipping
+===============================
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping.rst:5]: Isn't it a bit overkill to have a toctree page for a single sub-page?

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+Payment providers
+=================
+
+The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,
+adding additional payment providers for your customers is strongly recommended. Odoo supports a
+multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
+for your website.
+
+.. seealso::
+   :doc:`../../../../applications/finance/payment_providers`
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:11]:
```suggestion
   :doc:`../../../finance/payment_providers`
```
or
```suggestion
   :doc:`/applications/finance/payment_providers`
```

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 8]
```diff
@@ -0,0 +1,38 @@
+=================
+Payment providers
+=================
+
+The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:5]: Wire transfer is no longer installed by default in Odoo 16.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/websites/ecommerce/checkout_payment_shipping.rst (hunk 6 above)
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping.rst:5]: It's part of a bigger eCommerce doc overhaul https://docs.google.com/spreadsheets/d/1O7Jg4T8kPE0HaLw-3YDjaj54MMk957-9akBFaSNKQA0/edit#gid=0, it'll all be merged into one mega PR and this one will be closed then

Comment (AntoineVDV) [reply content/applications/websites/ecommerce/checkout_payment_shipping.rst:5]: Ok then, but I strongly advise *against* merging tons of changes at once with a single "mega PR". This has been tried over and over again, and it was always a bad idea :sweat_smile:

**Review by xpl-odoo — APPROVED** [review]

File: content/applications/websites/ecommerce/checkout_payment_shipping.rst [hunk 9]
```diff
@@ -0,0 +1,10 @@
+:nosearch:
```
Comment (xpl-odoo) [inline content/applications/websites/ecommerce/checkout_payment_shipping.rst:1]:
```suggestion
```
To be removed

**Comment by toaa-odoo** [comment]
@odoo/doc-review I'm aware of the conflict, I'll fix it before merging (other PRs are being merged right now, which may create further conflicts). Could you ignore that and have a look at the rest of the PR? Thanks!

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 10]
```diff
@@ -0,0 +1,37 @@
+=================
+Payment providers
+=================
+
+Odoo supports a multitude of online
+:doc:`payment providers <../../../../applications/finance/payment_providers>` for your website,
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:6]: My previous comment about unnecessary long refs applies to this `doc` role too.

File: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+available on your shop, and configure them according to your needs.
+
+Alternatively, you can access **payment providers** via :menuselection:`Website --> Configuration
+--> Settings`. In the :guilabel:`Shop - Payment` section, you can :guilabel:`Configure SEPA Direct
+Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
+:guilabel:`Authorize.net` payment provider, the
+:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
+same menu.
+
+If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure
```
Comment (AntoineVDV) [inline content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst:26]: Here too, but in this case it's correct like this if you prefer to use relative refs over absolute refs.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**